src/main/java/mate/academy/dao/impl/AbstractDao.java

```diff
@@ -33,8 +39,8 @@ public T add(T entity, Class<T> clazz) {
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
             return session.createQuery("from " + clazz.getSimpleName(), clazz).getResultList();
         } catch (Exception e) {
-            throw new DataProcessingException("Error retrieving all "
-                    + clazz.getSimpleName(), e);
+            logger.error("Cant fetch all {} from DB", clazz.getName(), e);
+            return Collections.emptyList();
```

Comment:
exceptions are logged by default - we should not log them
let's return to the original version here